docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/risk_effects/index.rst

```diff
@@ -35,3 +35,190 @@ Iron Deficiency Risk Effects
+
+.. image:: iron_risk_hierarchy.svg
+
+.. list-table:: Affected Entities
+   :widths: 5 5 5 5 5
+   :header-rows: 1
+
+   * - Outcome
+     - Outcome type
+     - Outcome ID
+     - Affected measure
+     - Note
+   * - Maternal disorders
+     - Cause
+     - 366
+     - YLDs/YLLs
+     - Maternal disorders is not a most-detailed cause; the iron deficiency risk factor affects each of the most-detailed causes in the maternal disorders cause group equally
+   * - Dietary iron deficiency
+     - Cause
+     - 390
+     - Prevalence
+     - PAF-of-1 relationship
+
+Maternal Disorders
+++++++++++++++++++
+
+The GBD 2019 Risk-Outcome pair relationship between iron deficiency and maternal disorders relies on the association between hemoglobin concentration and maternal mortality from the literature, specifically the CHERG iron report.
+
+.. todo::
+
+  Cite the CHERG iron report, GBD 2019 methods appendix
+
+.. note::
+
+  According to a correspondence with Chris Murray, there is contraversy surrounding the inclusion of this risk-outcome pair in GBD because there is evidence supporting an association between iron supplementation and hemoglobin increases as well as evidence supporting an association between hemoglobin increases and maternal mortality, but no/little evidence to support an association between iron supplementation and maternal mortality directly.
+
+Notably, the *same* relative risks for this risk-outcome pair apply to both YLDs and YLLs and are constant for each age-group and each most-detailed cause within the maternal disorders cause heirarchy; assuming that changes in hemoglobin concentration affect each of the maternal disorders equally and also affect morbidity and mortality equally.
+
+  A limitation of this approach is that if changes in hemoglobin concentration affect a maternal disorder with a higher disability weight more or less than a maternal disorder with a lower disability weight, then our estimation of change in maternal disorder YLDs due to a change in hemoglobin concentration will be biased. 
+
+.. todo::
+
+  Include fiugre for cause hierarychy of maternal disorders
+
+The relative risks used for this risk outcome pair are scaled to units of hemoglobin concentration in g/dL. Because the iron deficiency exposure (and mean population hemoglobin concentration) in GBD 2019 are defined in g/L, the units will need to be converted.
+
+The PAF for this risk outcome pair is computed using the :code-block:`cont_paf_nocap` function found `here <https://stash.ihme.washington.edu/projects/RF/repos/paf/browse/math.R#9-10,30>`_, with the following parameter values:
```

Comment:
I think `:code-block:` is only a directive, not a role, so the `cont_paf_nocap` function is not rendering correctly.   I think `:code:` might be a role, or you can just use double backticks, i.e. \``cont_paf_nocap``

Also, it looks like the `math.R` code has been updated since this PR was created, so the specific lines in the link seem to be pointing to the wrong place. In fact, I don't even see a `cont_paf_nocap` function, just a `cont_paf` function...

Would it make more sense to find the older version of the file and point to that, or update to point to the newer version (which could be subject to additional changes that could break the link again...)?